src/main/scala/org/apache/mesos/chronos/scheduler/jobs/JobScheduler.scala

```diff
@@ -609,10 +610,12 @@ class JobScheduler @Inject()(val scheduleHorizon: Period,
 
   //Begin Leader interface, which is required for CandidateImpl.
   def onDefeated() {
-    mesosDriver.close()
-
-    log.info("Defeated. Not the current leader.")
+    import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Might be worth commenting on why this is imported here, took me a while to figure out.